src/Adapter/Product/SpecificPrice/CommandHandler/EditSpecificPriceHandler.php

```diff
@@ -27,16 +27,18 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Product\SpecificPrice\CommandHandler;
 
+use PrestaShop\Decimal\DecimalNumber;
 use PrestaShop\PrestaShop\Adapter\Product\SpecificPrice\Repository\SpecificPriceRepository;
-use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\Command\EditProductSpecificPriceCommand;
-use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\CommandHandler\EditProductSpecificPriceHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\Command\EditSpecificPriceCommand;
+use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\CommandHandler\EditSpecificPriceHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\ValueObject\Reduction;
 use PrestaShop\PrestaShop\Core\Util\DateTime\DateTime;
 use SpecificPrice;
 
 /**
- * Handles @see EditProductSpecificPriceCommand using legacy object model
+ * Handles @see EditSpecificPriceCommand using legacy object model
  */
-class EditProductSpecificPriceHandler implements EditProductSpecificPriceHandlerInterface
+class EditSpecificPriceHandler implements EditSpecificPriceHandlerInterface
```

Comment:
Please define the BC Break

Reply:
It was never released, it isn't a BC Break. Same for a couple of other classes you mentioned.